src/tests/unit/tests/reports/components/report-sections/link-script-provider.test.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { isFunction } from 'lodash';
+import {
+    addEventListenerForLink,
+    getAddListenerForLink,
+} from 'reports/components/report-sections/link-script-provider';
+import { It, Mock, MockBehavior, Times } from 'typemoq';
+
+describe('LinkScriptProvider', () => {
+    describe('getAddListenerForLink', () => {
+        it('matches content', () => {
+            const code = 'this is test code';
+
+            const result = getAddListenerForLink(code);
```

Comment:
If we reuse the function from 'src\reports\components\report-sections\collapsible-script-provider.tsx', this test should be on a different place.

Reply:
Changed the function slightly, will be leaving the tests separate